### PR TITLE
Update best-practices.md

### PR DESCRIPTION
**Pull request by abhi1358**
## Why make this change?

- Improving the doc

## What is this change?

- Doc update

### CODE REVIEW

**Review by seantleonard — APPROVED** [review]
Thank you for your contribution 👍

File: docs/best-practices.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ When adding an entity to the configuration file, use PascalCasing, so that the g
   - `updateCompositeNameEntity`
   - `deleteCompositeNameEntity`
 
+- Stored Procedures
```
Comment (Aniruddh25) [inline docs/best-practices.md:15]: If the entity is backed by a stored procedure, both query or mutation would be named:
 - `executeCompositeNameEntity`

File: docs/best-practices.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ When adding an entity to the configuration file, use PascalCasing, so that the g
   - `updateCompositeNameEntity`
   - `deleteCompositeNameEntity`
 
+- If the entity is backed by a stored procedure, both query or mutation would be named:
```
Comment (seantleonard) [inline docs/best-practices.md:15]:
```suggestion
- If the entity is backed by a stored procedure, the generated query or mutation would be named:
```
nit: more explicit details.

**Review by seantleonard — CHANGES_REQUESTED** [review]
@abhi1358 , I believe other local changes made it into this PR for ignoring some nuget warnings and changing the package url. Those should be moved to separate PRs where not already created.

**Review by abhishekkumams — APPROVED** [review]

**Pull request merged**